bundles/org.openhab.binding.evohome/src/main/java/org/openhab/binding/evohome/internal/api/ApiAccess.java

```diff
@@ -89,18 +91,16 @@ public void setApplicationId(String applicationId) {
      * @return The result of the request or null
      * @throws TimeoutException Thrown when a request times out
      */
-    public <TOut> TOut doRequest(HttpMethod method, String url, Map<String, String> headers, String requestData,
-            String contentType, Class<TOut> outClass) throws TimeoutException {
-        TOut retVal = null;
+    public @Nullable <TOut> TOut doRequest(HttpMethod method, String url, Map<String, String> headers,
+            @Nullable String requestData, String contentType, @Nullable Class<TOut> outClass) throws TimeoutException {
         logger.debug("Requesting: [{}]", url);
-
+        @Nullable
```

Comment:
Is this annotation needed for local variable?

Reply:
When removed this warning appears:
`[WARNING] C:\oh-dev\openhab-addons\bundles\org.openhab.binding.evohome\src\main\java\org\openhab\binding\evohome\internal\api\ApiAccess.java:[98,23] Null type mismatch (type annotations): 'null' is not compatible to the free type variable 'TOut'`